src/style.css

```diff
@@ -0,0 +1,154 @@
+/* #region Normalize */
+
+body {
+  font-family: "Roboto", Helvetica, sans-serif;
+  margin: 0;
+  box-sizing: border-box;
+
+}
+/* #endregion */
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__product-img {
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+  margin-left: 19px;
```

Comment:
not a good practice to add`  margin-left `or  `margin-right `for every element
you can add `padding `and `align-items: center `to `card` class (parent) that will a align all your items in center and you dont need to add margins to every element
`padding: 32px 16px 16px; `to card will save lots of lines of code.

Reply:
Implemented. Thanks